src/Symfony/Component/Routing/Route.php

```diff
@@ -544,6 +533,24 @@ public function compile()
         return $this->compiled = $class::compile($this);
     }
 
+    private function extractInlineDefaultsAndRequirements(string $pattern): string
+    {
+        if (false !== strpbrk($pattern, '?<')) {
```

Comment:
if (false === etc) to use an early return clause?

Reply:
Better indeed, updated.